design/sequence/main.tex

```diff
@@ -0,0 +1,13 @@
+\chapter{Sequence diagrams}
+The following sequence diagrams describe the detailed interaction between individual classes of JobAdder:
+    \section{User client}
+        \includegraphics[width=\linewidth]{sequence/userclient-cli.png}
+    \section{Network communication}
+        \subsection{Oneshot message passing}
+            \includegraphics[width=\linewidth]{sequence/proxy-one-message.png}
+        \subsection{Continuous message passing}
+            \includegraphics[width=\linewidth]{sequence/proxy-continuous.png}
+        \subsection{Handling of remote commands on the server}
+            \includegraphics[width=\linewidth]{sequence/server-local-request.png}
+    \section{Rescheduling of jobs}
```

Comment:
finish-sd is not included in this, is that intentional?

Reply:
I have simply forgotten it.